src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	public int getTotal() {
+		return total;
+	}
+    private static final int MAX_TAB_SIZE = 12;
+    private static final int EMPTY = -1;
+
+    private final int[] numbers = new int[MAX_TAB_SIZE];
+
+    private int total = EMPTY;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.